hydrolib/core/dflowfm/tim/serializer.py

```diff
@@ -65,7 +68,9 @@ def _serialize_timeseries_lines(data, config) -> List[str]:
     @staticmethod
     def _serialize_to_timeseries_block(data, format_float) -> TimeSeriesBlock:
         timeseries_block: TimeSeriesBlock = []
-        for time, row_elements in data["timeseries"].items():
+        for nested_data in data["timeseries"]:
```

Comment:
What is `nested_data`? Can you come up with a word that better describes what is inside so we do not have to use `nested_data`? :D

Reply:
done